web/src/lib/utils/axios/axiosRetry.js

```diff
@@ -20,30 +20,47 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
-*/
-
-/**
- * Referred from src/utils/http/axios/axiosRetry.ts
  */
 
-import { AxiosError, AxiosInstance } from 'axios'
+var AxiosRetry = /** @class */ (function () {
+  function AxiosRetry() {}
+
+  AxiosRetry.prototype.retry = function (axiosInstance, error) {
+    var _a, _b;
+    var config = error.response.config;
 
-export class AxiosRetry {
-  retry(axiosInstance: AxiosInstance, error: AxiosError) {
-    const { config }: any = error.response
-    const { waitTime, count } = config?.requestOptions?.retryRequest ?? {}
-    config.__retryCount = config.__retryCount || 0
+    var _c =
```

Comment:
What do these codes represent? and I found similar anonymous functions elsewhere too.

Reply:
1. These codes handle the retry logic for Axios requests by extracting necessary retry parameters from the error response and managing retry counts. 
2. Similar anonymous functions are used throughout the codebase to simplify callback logic and ensure proper handling of asynchronous operations.
4. But by extracting named functions or using arrow functions, similar effects can also be achieved. 
6. **Do you want me to replace these anonymous functions with named functions or arrow functions?**

Reply:
I think using named functions or arrow functions is more readable and maintainable, what do you think?